components/alertdialog/stories/template.js

```diff
@@ -69,32 +70,29 @@ export const Template = ({
     </div>
   `;
 
-  return  html`
+  return html`
     ${Underlay({
       ...globals,
       isOpen,
+      onclick: () => {
+        if (isOpen) updateArgs({ isOpen: false });
+      },
     })}
     ${Button({
       ...globals,
       size: "m",
       variant: "secondary",
-      label: "Click to open Alert Dialog",
+      label: "Click to open",
```

Comment:
```suggestion
      label: "Click to open Alert Dialog",
```

I always prefer more explicit labels for accessibility reasons, just a suggestion

Reply:
Sure but context is relevant too. Screen readers offer context and we don't have to be too redundant, in part because it's kind of annoying when the reader keeps repeating itself. My thought here is, we're in the context of an Alert Dialog so click to open being the only action available is going to apply to this context.

Reply:
I'll also note, it's a theme that impacts every state and variant so adding it as just another item in the collection would make snapshots unmanageable to debug in my opinion.